content/deploy-troubleshooting.md

```diff
@@ -46,4 +61,19 @@ Version 1.2.1 of Meteor (and higher) provides the `METEOR_BINARY_DEP_WORKAROUND`
 
 If you are uncertain if this matches your situation, you can use [this test app](https://github.com/zol/meteor-bignum-test) to reproduce the error and confirm a fix.
 
+<h2 id="memory-issues">Memory issues or Out-of-Memory errors</h2>
+
+Memory issues are frequently indicated in your logs. The log message `The container has run out of memory. A new container will be started to replace it.` means that the container running your application tried to get more memory than was allocated to it. Containers in this state are automatically killed.
+
+You can see memory utilization in the container view of your app. Check the 5 minute, 1 hour, 1 day and 30 day views; if you see memory spiking to 90% or more, that is almost certainly the problem.
+
+If your container is running out of memory, you may need to [scale up](/scaling.html) to fix this. You can always scale back down after refactoring your app to consume less memory. If you continue to run your app as is, the only way to prevent it from dying is to allocate containers with more than enough memory.
```

Comment:
again, it's possible that more containers rather than bigger might work too.  depends if it's "need to be able to deal with big spikes occasionally" vs "each connection takes X memory, we have lots of connections".